### PR TITLE
implement neighbourhood exploration iterables from python

### PR DESCRIPTION
**Pull request by ljeub-pometry**
Implements recursive exploration of neighbours from the Python API

### CODE REVIEW

File: pyraphtory/src/graph_window.rs [hunk 1]
```diff
@@ -38,11 +40,15 @@ impl WindowedGraph {
...
-            iter: Box::new(self.graph_w.vertices().map(|wv| wv.into())),
+    pub fn vertices(slf: PyRef<'_, Self>) -> WindowedVertexIterable {
+        let g: Py<Self> = slf.into();
+        WindowedVertexIterable {
+            graph: g,
+            operations: vec![],
+            start_at: None,
         }
     }
+    // graph.vertices().out_neighbours().map(|v| v.degree()).sum())
```
Comment (fabianmurariu) [inline pyraphtory/src/graph_window.rs:51]: remove?

**Review by fabianmurariu — APPROVED** [review]
LGTM

File: pyraphtory/src/graph_window.rs (hunk 1 above)
Comment (ljeub-pometry) [reply pyraphtory/src/graph_window.rs]: done

**Review by fabianmurariu — APPROVED** [review]
LGTM

**Pull request merged**